### ISSUE TITLE
[Bug]: latest version available on windows is 0.2.0

### ISSUE THREAD

**Issue by matteodefelice** [xCDAT/xcdat#342]
### What happened?

I have created a new environment with the command `conda create -n xcdat -c conda-forge xcdat` and the installed xcdat version is `xcdat              conda-forge/noarch::xcdat-0.2.0-pyhd8ed1ab_0`.

If I ask anaconda to install the 0.3.1, it says that `esmpy` is not available. 
```
- nothing provides esmpy needed by xcdat-0.3.1-pyhd8ed1ab_0
```

### Environment

.

**Comment by pochedls** [comment]
@matteodefelice – thanks for reporting this. I'm kind of confused – I don't have `xesmpy` in my `conda list` for xcdat 0.3.1 (though this is on Linux). Maybe this is a xesmf-windows-dependency? 

@durack1 hit what seems like a [related issue](https://github.com/xCDAT/xcdat/issues/315) on the new ARM macbooks; xesmf doesn't yet have a build, so @durack1 has a PR that makes xesmf an optional import in xcdat. I'm guessing this will be merged and incorporated into 0.3.2 in the next few weeks.

**Comment by durack1** [comment]
@pochedls correct, `esmpy` (not `xesmpy`) is only available for `linux-64` and `osx-64` currently, see [here](https://anaconda.org/conda-forge/esmpy).

It is likely that [`osx-arm64` support will appear in an upcoming `esmpy=8.3/.4` release](https://github.com/conda-forge/esmf-feedstock/pull/75#issuecomment-1230706928). I do not believe that `windows` support is currently planned, however [an issue exists](https://github.com/conda-forge/esmf-feedstock/issues/64) - @matteodefelice you may like to comment on that open issue to add support for `windows`

**Comment by durack1** [comment]
@matteodefelice, another option is to use the [Windows native subsystem for linux](https://docs.microsoft.com/en-us/windows/wsl/install) and then use the `linux-64` binaries in their native environment on windows.

@tomvothecoder this approach might be a useful thing to document in readthedocs

**Comment by tomvothecoder** [comment]
@durack1, it looks like Matteo is using WSL2 as an alternative (https://github.com/xCDAT/xcdat/issues/341).

That's a good workaround for Window users, so it will be included in #334.

**Comment by tomvothecoder** [comment]
I'll close this issue since #315 covers this too.
